app/components/drag-handle.js

```diff
@@ -10,30 +10,29 @@ export default Ember.Component.extend({
   isLeft: Ember.computed.equal('side', 'left'),
   minWidth: 60,
 
-  startDragging: function() {
-    let self = this,
-        $container = this.$().parent(),
+  startDragging() {
+    let $container = this.$().parent(),
```

Comment:
These seem like it could be `const`? I don't see the reference changing during the scope.